src/Chapter3.hs

```diff
@@ -560,22 +678,31 @@ introducing extra newtypes.
 🕯 HINT: if you complete this task properly, you don't need to change the
     implementation of the "hitPlayer" function at all!
 -}
+
+newtype Healt = Healt Int
+newtype Armor = Armor Int
+newtype Attac = Attac Int
```

Comment:
I see that you renamed these newtypes to avoid conflicts with one of the first task. If you want, you can actually reuse these newtypes in Task 4. Also, no need to move the code around, in Haskell any definition inside a module is visible in the whole module